### DISCUSSION TITLE
Mark certain strings as secret in `across`

### DISCUSSION THREAD

**Discussion by killermoehre** [concourse/concourse#8521] [Help & Support]
Hi,

I use `set_pipeline` to create pipelines. In this process I transfer certain secrets into the new pipeline.
In the new pipeline I use a `across` step, which shows the secret, as it is one of it inputs.

Can I somehow mark this string as a secret? I know I can do this in GH actions.

**Comment by marco-m** [comment]
See https://concourse-ci.org/creds-redacting.html

**Reply by killermoehre** [reply]
Well, I use

```yaml
- across:
    - var: region
      values: ((regions))
```

where `regions` is a list of elements read from a config file. In the config file, the password is listed as `((vault:some/path))`.

And the output of `across` shows always all parameters provided.

**Reply by marco-m** [reply]
It is unclear if you enabled credentials redaction or not.

**Reply by killermoehre** [reply]
It is enabled.
